hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -25,6 +25,8 @@ import (
 	"time"
 
 	"github.com/openshift/hypershift/api"
+	"github.com/openshift/hypershift/control-plane-operator/controllers/hostedcontrolplane/kas"
+	kasmanifests "github.com/openshift/hypershift/control-plane-operator/controllers/hostedcontrolplane/manifests"
```

Comment:
This is an illegal dependency — the control-plane-operator is versioned with OCP payloads and there should be no Go dependencies on it. The definition for the secret as managed by hypershift-operator belongs in the `controllers/manifests/controlplaneoperator` package.

Reply:
so would you rather the secret be duplicated in two locations? I can do that

Reply:
This actually totally makes sense I see what you mean

Reply:
Believe I fixed